client/src/app/learn/learn.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-learn',
+  templateUrl: './learn.component.html',
+  styleUrls: ['./learn.component.scss']
+})
+export class LearnComponent implements OnInit {
+  wordList: any[] = [
+    {
+      'en': 'Dance',
+      'pl': 'Tańczyć'
+    },
+    {
+      'en': 'Talk',
+      'pl': 'Mówić'
+    },
+    {
+      'en': 'Walk',
+      'pl': 'Chodzić'
+    },
+  ];
+  currentId: number = 0;
+  lang: string = 'en';
+  secondLang: string = 'pl';
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  toggleLanguage(): void {
+    let lang = this.lang;
+    this.lang = this.secondLang;
+    this.secondLang = lang;
+  }
+
+  changeWord(num = 1): void {
+    let nextId = this.currentId + num;
+
+    if (this.wordList[nextId]) this.currentId += num;
+  }
+
+  onSubmit(value): void {
```

Comment:
Zostawiłem przy sprawdzeniu, bo wychodzę z założenia, że ten komponent nie wie na 100% w jakiej formie dostanie wyrazy. Jeżeli potem okaże się zbędne to zmienię.